prescriptions/_containers/ps_cv_pytorch/recommendations.yaml

```diff
@@ -90,3 +90,35 @@ units:
       - type: INFO
         message: Consider using a computer vision predictable stack 'ps-cv-pytorch' that has prepared environment with 'jupyter-tensorboard'
         link: https://quay.io/repository/thoth-station/ps-cv-pytorch
+
+  wraps:
+  - name: ThothPSCVTorch190BaseImageWrap
+    type: wrap
+    should_include:
+      adviser_pipeline: true
+      runtime_environments:
+        operating_systems:
+        - name: rhel
+          version: "8"
+        python_version: ==3.8
+        base_images:
+          not:
+          - quay.io/thoth-station/ps-cv-pytorch
+    match:
+      state:
+        resolved_dependencies:
+        - name: torch
+          version: ==1.9.0
+        - name: opencv-python
+          version: ==4.5.2.54
+    run:
+      stack_info:
+      - type: INFO
+        message: >-
+          Found predictive stack image that can be used with these dependencies
+        link: quay.io/thoth-station/ps-cv-pytorch:v0.1.2
+      advised_manifest_changes:
```

Comment:
We do not have support for these `advised_manifest_changes` yet.

Reply:
Do you mean from thamos side?

Reply:
hey @fridex, support for `advised_manifest_changes` is a feature we need to add to adviser?

Reply:
Some parts were discussed in https://github.com/thoth-station/thamos/issues/910 Feel free to summarize efforts in a separate issue if you want.

Reply:
https://github.com/thoth-station/adviser/issues/2211 :)